packages/web-app-files/src/views/PublicFiles.vue

```diff
@@ -142,6 +135,10 @@ export default {
     }
   },
 
+  data: () => ({
+    loading: true
```

Comment:
`loading` is not referenced, right? Then you can delete it again. dev leftover?

Reply:
yeah it is, its for the list-loader